bundles/pcmjava/tools.vitruv.applications.pcmjava.common/src/tools/vitruv/applications/pcmjava/common/pcm2java/Pcm2JavaCommon.reactions

```diff
@@ -520,8 +548,9 @@ routine createJavaPackage(EObject sourceElementMappedToPackage, java::Package pa
 			javaPackage.name = packageName.toFirstLower;
 			persistProjectRelative(sourceElementMappedToPackage, javaPackage, buildJavaFilePath(javaPackage));
 		}
-		add correspondence between javaPackage and sourceElementMappedToPackage
-			tagged with newTag
+		add correspondence between javaPackage and sourceElementMappedToPackage tagged with newTag
+		// Required to enable locating existing packages with missing correspondences when keeping more than two models consistent:
+		add correspondence between javaPackage and javaPackage.eClass
```

Comment:
Although `ContainersPackage.Literals.PACKAGE` (used in retrieval) and `javaPackage.eClass` (used in creation) should always be equal, it would be beneficial to always use `ContainersPackage.Literals.PACKAGE` when creating and retrieving the same correspondence to improve comprehensibility.